### PR TITLE
Fix: Added Responsiveness in product page

### PR DESCRIPTION
**Pull request by sharathnair9999**
Added Filter Button in mobile view for Products page to toggle to the filter section and come back to the products page. Look at the preview and give your review.

### CODE REVIEW

**Review by biswanath-tewari — APPROVED** [review]

**Review by SukanyaSen530 — APPROVED** [review]

File: styles/style.css [hunk 1]
```diff
@@ -44,11 +44,12 @@ body{
     position: relative;
     clear: both;
     width: 100%;
+    height: 5rem;
     padding: 0.5rem 1rem;
     display: flex;
     justify-content: space-around;
     align-items: center;  
-    background:var(--primary-500);
+    background:#a5f3fc;
```
Comment (VedanthB) [inline styles/style.css:52]: why arent you using css var here ?

File: styles/Products/products.css [hunk 2]
```diff
@@ -50,7 +76,7 @@
   justify-content: space-around;
   align-items: center;
   padding: 2px 8px;
```
Comment (VedanthB) [inline styles/Products/products.css:78]: use rem here, be consistent with the units you use throughout your project

**Review by VedanthB — COMMENTED** [review]
everything looks good to go 🚀

File: styles/style.css (hunk 1 above)
Comment (sharathnair9999) [reply styles/style.css:52]: oh didnt see that one left out. will do it right away

**Pull request merged**